html/archived_changelog/changelog.html

```diff
@@ -25,7 +25,8 @@
 		<td valign='top'>
 			<div align='center'><font size='3'><b>Paradise Station</b></font></div>
 			
-			<p><div align='center'><font size='3'><a href="https://www.paradisestation.org/forum/">Forum</a> | <a href="https://www.paradisestation.org/wiki/index.php/Main_Page">Wiki</a> | <a href="https://github.com/ParadiseSS13/Paradise">Source</a></font></div></p>
+			<p><div align='center'><font size='3'><a href="https://www.paradisestation.org/forum">Forum</a> | <a href="https://www.paradisestation.org/wiki/index.php/Main_Page">Wiki</a> | <a href="https://github.com/ParadiseSS13/Paradise">Source</a></font></div></p>
+			<font size='2'><b>Visit our Discord channel:</b><a href='https://discordapp.com/invite/nuqD478<'>-Click Here-</a></font>

```

Comment:
Remove the Discord invite link. It is outdated and the actual link changes every once in awhile, we don't want to have static links in the HTML.